### ISSUE TITLE
tutorials may become broken after next scikit-learn release

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#179]
In the tutorials we rely on the fact classifier and vectorizer are fit after a pipeline is fit - see e.g. http://eli5.readthedocs.io/en/latest/tutorials/sklearn-text.html. But this is changed in scikit-learn development version, so the tutorials should be updated to make no use of this pattern. 

Maybe it already became easier after https://github.com/TeamHG-Memex/eli5/pull/177, but we need to check that, and it also needs to be fixed for explain_prediction. Alternative is to avoid using Pipeline in these tutorials, or use pipe.named_steps instead of original clf/vec.

**Comment by jnothman** [comment]
I don't think this is changed in current scikit-learn master, is it? That's
a backward incompatibility we can't afford.

**Comment by kmike** [comment]
Oh, right, for some reason I thought https://github.com/scikit-learn/scikit-learn/pull/8350 is already merged and that it is not backwards compatible, while both of it is false. Thanks for the catch!
